docs/help-center/tim-features/reset-password.mdx

```diff
@@ -0,0 +1,64 @@
+---
+sidebar_label: Reset password
+title: Reset password
+hide_title: true
+displayed_sidebar: helpcenterSidebar
+sidebar_position: 5
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+
+<div class="hero hero--primary">
+  <div class="container">
+    <h2 class="hero__title">Reset Password</h2>
+    <p class="hero__subtitle">How to reset your password if you have forgotten it or want to change it.</p>
+  </div>
+</div>
+
+## Change password
+
+To change the password, please follow [these steps](../your-account/account-settings#change-password).
+
+## Reset forgotten password
+
+:::info
+You will only be able to reset your password this way if your account is on a TI-Messenger server. If no new browser window or tab opens for login, you are most likely not on a TI-Messenger server and need to contact your administrator for resetting your password.
+:::
+
+<Tabs>
+  <TabItem value="desktop" label="Desktop">
+  <ol>
+    <li>Open Famedly web app.</li>
+    <li>Enter your login URL into the corresponding input field.</li>
+    <li>Click <b>Next</b>.</li>
+    <li>A new tab for entering the login credentials will open.</li>
+    <li>Enter your username and click **Continue**.</li>
+    <li>Click <b>Reset password</b>.</li>
+    <li>Check your mails for a mail with the title <b>Reset password</b>.</li>
```

Comment:
Check your email inbox for ... 
We should probably also mention the Spam folder here.